backend/src/test/java/hanglog/expense/presentation/ExpenseControllerTest.java

```diff
@@ -0,0 +1,203 @@
+package hanglog.expense.presentation;
+
+import static hanglog.category.fixture.CategoryFixture.EXPENSE_CATEGORIES;
+import static hanglog.expense.fixture.CurrenciesFixture.DEFAULT_CURRENCY;
+import static hanglog.trip.fixture.CityFixture.LONDON;
+import static hanglog.trip.fixture.CityFixture.TOKYO;
+import static hanglog.trip.fixture.ItemFixture.LONDON_EYE_ITEM;
+import static hanglog.trip.fixture.ItemFixture.TAXI_ITEM;
+import static hanglog.trip.fixture.TripFixture.LONDON_TRIP;
+import static hanglog.trip.restdocs.RestDocsConfiguration.field;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+
+import hanglog.expense.dto.response.ExpenseGetResponse;
+import hanglog.expense.service.ExpenseService;
+import hanglog.trip.domain.DayLog;
+import hanglog.trip.domain.TripCity;
+import hanglog.trip.restdocs.RestDocsTest;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.jpa.mapping.JpaMetamodelMappingContext;
+import org.springframework.restdocs.payload.JsonFieldType;
+
+@WebMvcTest(ExpenseController.class)
+@MockBean(JpaMetamodelMappingContext.class)
+class ExpenseControllerTest extends RestDocsTest {
+
+    @MockBean
+    private ExpenseService expenseService;
+
+    @DisplayName("모든 경비를 가져온다.")
+    @Test
+    void getExpenses() throws Exception {
+        // given
+        final DayLog LONDON_DAY = new DayLog(1L, "런던추워", 1, LONDON_TRIP, List.of(LONDON_EYE_ITEM, TAXI_ITEM));
+        final ExpenseGetResponse expenseGetResponse = ExpenseGetResponse.of(
+                LONDON_TRIP,
+                10000,
+                List.of(new TripCity(LONDON_TRIP, LONDON), new TripCity(LONDON_TRIP, TOKYO)),
+                Map.of(EXPENSE_CATEGORIES.get(1), 1000),
+                DEFAULT_CURRENCY,
+                Map.of(LONDON_DAY, 1000)
+        );
+
+        // when & then
+        when(expenseService.getAllExpenses(1L)).thenReturn(expenseGetResponse);
+
+        mockMvc.perform(get("/trips/{tripId}/expense", 1).contentType(APPLICATION_JSON))
+                .andDo(
+                        restDocs.document(
+                                responseFields(
```

Comment:
RestDocs 적용 👍👍

위에도 남겼지만 테스트에도 pathPatamter 추가해주세요~~
```suggestion
                        restDocs.document(
                                pathParameters(
                                        parameterWithName("tripId")
                                                .description("여행 ID")
                                ),
                                responseFields(
```